components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -85,6 +85,10 @@ export class Workspaces extends React.Component<WorkspacesProps, WorkspacesState
                         onClick: () => { if (wsModel) wsModel.limit = 200; }
                     }]} />
                 </div>
+                {wsModel && this.state?.workspaces.length > 0 ? 
+                 <button onClick={toggleStartWSModal} className="ml-2 font-medium">Start Workspace</button>
```

Comment:
**suggestion**: Same as with [the button on the empty state](https://github.com/gitpod-io/gitpod/pull/3539/files#diff-4a5c8bb479a125673053c7e43fe340a3bb1ffb478115a0003ae497c5005a1dcdR139).

```suggestion
                 <button onClick={toggleStartWSModal} className="ml-2 font-medium">New Workspace</button>
```